config/environments/test.rb

```diff
@@ -35,7 +35,7 @@
 
   # Tests assume English text on the site.
   config.i18n.default_locale = "en"
-  config.i18n.available_locales = ['en', 'es']
+  config.i18n.available_locales = ['en', 'es', 'pt']
```

Comment:
actually we need 'de' here as well to fix spec/features/admin/order_cycles_spec.rb:123:
:-)

Reply:
I leave that one :point_up: to you @Matt-Yorkley. Feel free to merge afterwards.

Reply:
Isso mudou... 

I changed that test in `spec/features/admin/order_cycles_spec.rb:123` from `:de` to `:pt` and now it's green.

Reply:
:+1: